### ISSUE TITLE
Fonts missing when building for production

### ISSUE THREAD

**Issue by n1try** [jens-ox/prm#14]
Everything works fine when using `yarn serve` (left in the screenshot). However, when trying to build a dist package with `yarn build` and serving it as static files, fonts don't seem to be loaded (right in the screenshot).

![Screenshot from 2020-03-03 14-13-59](https://user-images.githubusercontent.com/8978843/75779100-55223880-5d59-11ea-850e-c2dc7236027e.png)

**Comment by jens-ox** [comment]
This is likely caused by purgeCSS, I'll look into it later 😊

**Comment by n1try** [comment]
You're right. Problem occurs when doing `NODE_ENV=production yarn build`, but not for `NODE_ENV=dev yarn build`

**Comment by jens-ox** [comment]
Also notice the ugly body margin enforced by the default user stylesheet in the second screenshot 😅 
Implemented exactly the same yesterday in another project: https://github.com/jens-ox/metrics-graphics/commit/b50f07859e6a519aa06349f25868983a88923ecb

**Comment by jens-ox** [comment]
Updated in 572f8dc41847a3701b5f67348a40904b72be3a01, could you re-check?

**Comment by muety** [comment]
Front page is fine now, but the activity chart still looks a bit distorted.
![Screenshot from 2020-04-10 10-22-41](https://user-images.githubusercontent.com/8978843/78975847-4fb4cc80-7b15-11ea-9a84-a2d8ffd9a03e.png)
